scripts/module/ghostcars.nut

```diff
@@ -39,3 +36,9 @@ function onPlayerExitVehicle( player, vehicle ) {
     vehicle.IsGhost = false;
   }
 }
+
+function onPlayerPart( player, reason ) {
+  if (player.Vehicle != null) {
+    player.Vehicle.IsGhost = false;
+  }
```

Comment:
This probably also triggers for passengers

Reply:
Oh, of course. Fixed!